### PR TITLE
vdk-jupyter: UI failing cell marks enhancement

### PR DESCRIPTION
**Pull request by duyguHsnHsn**
What:
Added more colouring to the cell 
See the numbering on the left side:
Now:
Screenshot 2023-05-26 at 9 27 38
Then: 
Screenshot 2023-05-10 at 18 41 51

Why: we received feedback form users that it would be more user-friendly to have that part of the cell coloured on fail, too

Signed-off-by: Duygu Hasan [hduygu@vmware.com](mailto:hduygu@vmware.com)

### CODE REVIEW

File: projects/vdk-plugins/vdk-jupyter/vdk-jupyterlab-extension/src/components/RunJob.tsx [hunk 1]
```diff
@@ -160,9 +176,13 @@ export const findFailingCellInNotebookCells = async (
...
-      i === failingCellIndex
-        ? switchToFailingCell(cells[i])
-        : cells[i].classList.remove('jp-vdk-failing-cell');
+      if (i === failingCellIndex) {
+        switchToFailingCell(cells[i]);
+      } else {
+        cells[i].classList.remove('jp-vdk-failing-cell');
+        const cellPropt = getCellInputAreaPromp(cells[i]);
+        cellPropt?.classList.remove('jp-vdk-failing-cell-prompt')
+      }
```
Comment (ivakoleva) [inline projects/vdk-plugins/vdk-jupyter/vdk-jupyterlab-extension/src/components/RunJob.tsx:185]: nit: I would consider having the two functions to mark/unmark a cell as failing, then eval those. Think about the interface surface.

File: projects/vdk-plugins/vdk-jupyter/vdk-jupyterlab-extension/src/components/RunJob.tsx [hunk 2]
```diff
@@ -123,7 +123,23 @@ export const findFailingCellId = (message: String): string => {
   return '';
 };
 
+export const getCellInputAreaPromp = (failinCell: Element): Element | undefined => {
```
Comment (ivakoleva) [inline projects/vdk-plugins/vdk-jupyter/vdk-jupyterlab-extension/src/components/RunJob.tsx:126]: nit: may you document the purpose of this function? It's not very clear when reading the definition.

**Review by ivakoleva — COMMENTED** [review]
LGTM

**Review by ivakoleva — APPROVED** [review]

File: projects/vdk-plugins/vdk-jupyter/vdk-jupyterlab-extension/src/components/RunJob.tsx (hunk 2 above)
Comment (duyguHsnHsn) [reply projects/vdk-plugins/vdk-jupyter/vdk-jupyterlab-extension/src/components/RunJob.tsx:126]: done

File: projects/vdk-plugins/vdk-jupyter/vdk-jupyterlab-extension/src/components/RunJob.tsx (hunk 1 above)
Comment (duyguHsnHsn) [reply projects/vdk-plugins/vdk-jupyter/vdk-jupyterlab-extension/src/components/RunJob.tsx]: done

**Review by murphp15 — APPROVED** [review]

**Pull request merged**